Modules/Phasing.lua

```diff
@@ -666,15 +666,15 @@ function Phasing.IsSpawnVisible(phase)
     end
 
     if phase == phases.RHEA_NEW_KARGATH then
-        return complete[27888] and (not complete[27897]) and (not questLog[27897]) or false
+        return complete[27888] and ((not complete[27897]) and ((not questLog[27897]) or (questLog[27897] and questLog[27897].isComplete == 0))) or false
```

Comment:
27897 does not have any objectives so `questLog[27897].isComplete == 0` can never be `true`. So I am sure this can be removed again.